draft-ietf-quic-transport.md

```diff
@@ -2960,7 +2960,7 @@ data to a peer.
 
 ~~~
        o
-       | Application Open
+       | Open Stream
        | Open Paired Stream (bidirectional)
        v
```

Comment:
The term "Paired Stream" only appears in the figures. Shouldn't this be Open Unidirectional Stream, Open Bidirectional Stream?

Reply:
Good point.  Does the latest change work for you?

Reply:
yeah - sort of - Receive in Send diagram is a bit confusing - but I suppose it means also get ready to receive data.